pkg/cloudprovider/providers/gce/gce_alpha.go

```diff
@@ -28,7 +28,7 @@ var knownAlphaFeatures = map[string]bool{
 }
 
 const (
-	GCEDiskAlphaFeatureGate = "GCEDiskAlphaAPI"
+	GCEDiskAlphaFeatureGate = "DiskAlphaAPI"
```

Comment:
This is deliberate, right?

Reply:
Yep, I received a comment offline saying that this was redundant.